.github/workflows/playwright.yml

```diff
@@ -0,0 +1,54 @@
+name: E2E tests - Playwright
+on: [pull_request]
+
+jobs:
+  playwright-tests:
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        GRAFANA_VERSION: ['latest', '10.1.5', '10.0.6', '9.5.0', '9.1.5', '9.0.5', '8.5.5']
```

Comment:
Rather than having target Grafana versions hardcoded like this, we'd eventually want a script that resolves versions according to some rule that plugin teams have agreed upom. For example, "canary + last six minors" or "canary + all minors in current major + last minor in previous major).

Reply:
This would be great! I like this idea 🙌🏻